### PR TITLE
[airthings] Add support for Airthings Wave Gen 1

### PR DESCRIPTION
**Pull request by dw-8**
Add support for Airthings Wave Gen 1

Signed-off-by: Davy Wong davy.wong.on+github@gmail.com

### CODE REVIEW

File: bundles/org.openhab.binding.bluetooth.airthings/src/main/resources/OH-INF/thing/airthings.xml [hunk 1]
```diff
@@ -69,6 +69,30 @@
 			</parameter>
 		</config-description>
 	</thing-type>
+	<thing-type id="airthings_wave_gen1">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="roaming" />
+			<bridge-type-ref id="bluegiga" />
+			<bridge-type-ref id="bluez" />
+		</supported-bridge-type-refs><label>Airthings Wave Gen 1</label><description>Smart Radon Monitor</description><channels>
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.bluetooth.airthings/src/main/resources/OH-INF/thing/airthings.xml:77]: Some line breaks seem to be missing here.
Please run a `mvn spotless:apply`.

File: bundles/org.openhab.binding.bluetooth.airthings/src/main/java/org/openhab/binding/bluetooth/airthings/internal/AirthingsWaveGen1Handler.java [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+    @Override
+    protected void updateChannels(int[] is) {
+        int[] rawdata;
+        rawdata = is;
+        if (rawdata.length == 2) {
+            switch (readSensor) {
+                case TEMPERATURE:
+                    dblResult = intFromBytes(rawdata[0], rawdata[1]) / 100D;
+                    logger.info("Parsed data 1: {}", String.format("[temperature=%.1f °C]", dblResult));
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.bluetooth.airthings/src/main/java/org/openhab/binding/bluetooth/airthings/internal/AirthingsWaveGen1Handler.java:66]: Please only use debug level for all those logs (here and below).

**Review by kaikreuzer — CHANGES_REQUESTED** [review]
Thanks for this addition!
Just some very small review comments before we can merge.

File: bundles/org.openhab.binding.bluetooth.airthings/src/main/resources/OH-INF/thing/airthings.xml (hunk 1 above)
Comment (dw-8) [reply bundles/org.openhab.binding.bluetooth.airthings/src/main/resources/OH-INF/thing/airthings.xml]: I made the corrections.

File: bundles/org.openhab.binding.bluetooth.airthings/src/main/java/org/openhab/binding/bluetooth/airthings/internal/AirthingsWaveGen1Handler.java (hunk 2 above)
Comment (dw-8) [reply bundles/org.openhab.binding.bluetooth.airthings/src/main/java/org/openhab/binding/bluetooth/airthings/internal/AirthingsWaveGen1Handler.java]: I changed all logging to debug level only, as requested.

**Comment by dw-8** [comment]
I messed up this PR.

I merged openhab:main into this PR and then reverted that commit, but these other commits are still showing as files being changed in this PR, as if I am reverting them all out of main.

Do I need to revert the revert to fix this?

I am new to using github, and getting myself all confused when and where to sync (fetch and merge) forks and pull requests.

**Comment by dw-8** [comment]
I reverted the revert, and it correctly showed only my files being changed, so then I issue the "git rebase HEAD~# --signoff" to clear the DCO signoff requirements, but, then all the other users' individual commits (instead of the merge commit) all came back into my PR again.

I reverted all those users' commits, and then issued another "git rebase HEAD~# --signoff" but it still shows extra files being changed.

What am I doing wrong?

**Comment by dw-8** [comment]
Well, making those revisions to two files became very messy with a lot of extra commits...
I accidentally merged a bunch of other commits into my PR, which I tried to revert and then revert them again.
I think I cleaned it all up, but now I have 3 extra reviewers because of those other commits.

**Comment by dw-8** [comment]
@kaikreuzer Are you okay with the changes I made in address your requested changes?

**Review by kaikreuzer — APPROVED** [review]
Yes, looks good to me, thank you!

**Comment by kaikreuzer** [comment]
FTR, the formatting was still not ok, I have created https://github.com/openhab/openhab-addons/pull/11068 to fix it.

**Comment by kaikreuzer** [comment]
FTR, the SAT tests were failing as well 😢 . Fixed it with https://github.com/openhab/openhab-addons/pull/11074.

**Pull request merged**